lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -415,8 +415,9 @@ setThunk target gs branch = do
 -- This tool will only ever produce the newest one when it writes a thunk.
 gitHubStandaloneLoaders :: NonEmpty Text
 gitHubStandaloneLoaders =
-  gitHubStandaloneLoaderV2 :|
-  [ gitHubStandaloneLoaderV3
+  gitHubStandaloneLoaderV4 :|
+  [ gitHubStandaloneLoaderV2
+  , gitHubStandaloneLoaderV3
```

Comment:
Might as well order them?

Reply:
Oh I see why you did this. Once it's not the default order doesn't matter so might as well order them IMO to prevent anyone thinking there's a reason for this.